app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class QuestionsController < ApplicationController
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.new(question_params)
+    if @question.save
+      flash[:success] = 'Question successfully added'
+      redirect_to questions_path
+    else
+      flash[:warning] = @question.errors.full_messages.join(', ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.